sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -1175,12 +1180,22 @@ void initReduLocalAccs(size_t LID, size_t GID, size_t NWorkItems, size_t WGSize,
                        std::tuple<LocalAccT...> InputAccs,
                        const std::tuple<ResultT...> Identities,
                        std::index_sequence<Is...>) {
+  // Normally, the local accessors are initialized with elements from the input
+  // accessors. The exception is the case when (GID >= NWorkItems), which
+  // possible only when UniformPow2WG is false. For that case the elements of
+  // local accessors are initialized with identity value, so they would not
+  // give any impact into the final partial sums during the tree-reduction
+  // algorithm work.
   if (UniformPow2WG || GID < NWorkItems)
```

Comment:
```suggestion
  if (Pow2WG || GID < NWorkItems)
```

Reply:
Why do you suggest replacing it here? The template parameter of the function is UniformPow2WG, it has a bit different meaning for Aux kernel reductions than for main reduction kernels.